index.bs

```diff
@@ -204,14 +204,18 @@ NOTE: The dimensions indicated in the 'ispe' property might not match the values
 
 NOTE: The values of [=render_width_minus1=] and [=render_height_minus1=] possibly present in the AV1 bistream are not exposed in the AVIF container level.
 
+<h4 id="clean-aperture-property">Clean Aperture Property</h4>
+
+The semantics of the clean aperture property ('clap') as defined in [[!HEIF]] apply. In addition to the restrictions on transformative item property ordering specified in [[!MIAF]], the following restriction also applies:
+
+<assert>The origin of the 'clap' item property shall be anchored to 0,0 (top-left) of the input image unless the full, un-cropped image item is included as a secondary non-hidden image item.</assert>
```

Comment:
Good point.

From HEIF:

> A hidden image item has (flags & 1) equal to 1 in its ItemInfoEntry. 

So that it is easy to check.  
The presence of a `clap` property and whether it covers the full image should be straightforward too.

However an image item can be hidden or not, and cropped or not, but these are properties tied to the item, so I do not see how to have a cropped version and a non-cropped version of the same image item. Is it why collections were mentioned at first? Can one crop a collection? I guess it could be done through `iden` for example but checking if the full image is available through derived image items sounds a bit of work.  
Should it be "two image items referring to the same coded image bytes"?

Reply:
> I guess it could be done through iden for example but checking if the full image is available through derived image items sounds a bit of work.
> Should it be "two image items referring to the same coded image bytes"?

The way I envision someone doing this is per NOTE 2 in HEIF section 6.6.2.1 Identity derivation:
```
NOTE 2 A derived image of type 'iden' can be used, for example, when it is desirable to
have both the original version of a coded image item and a cropped version of the same
coded image item (obtained through the 'clap' transformative item property) as non-
hidden image items.
```

There are other ways it could be done, but this feels like the most straight forward and is relatively easy for a parser to check.

If someone uses an alternative scheme to do it, I think that should be allowed by the spec. It might not necessarily be implemented in all parsers, but that is something that can later be implemented if it turns out to be useful.